Articles/Setup/Install/Unattended-Install.md

```diff
@@ -0,0 +1,124 @@
+
+You can use the **Unattended installs** feature to allow for quick installation and set up of Umbraco instances on something like Azure Web Apps.
+
+This article will give you the details you need to install Umbraco unattended.
+
+## Get the correct version of Umbraco
+
+Since its beginning **Umbraco version 9** supports this feature.
+
+So it will be enough to get a clean instance from either the [NuGet feed](https://www.nuget.org/packages/UmbracoCms/) or download a zip file directly from [Downloads](https://our.umbraco.com/download).
+
+## Configure your database
+
+As you will not be running through the installation wizard when using the unattended installs feature, you need to manually tell Umbraco which database to use.
+
+* Set up and configure a new database - see [Requirements](../Requirements/#hosting) for details.
+* Add the connection string using configuration.
+
+Example in appsettings.json
+
+```json
+{
+  "ConnectionStrings": {
+    "umbracoDbDSN": "server=localhost;database=UmbracoUnicore;user id=sa;password='P@ssw0rd'"
+  }
+}
+```
+
+## Enable the unattended installs feature
+
+The unattended installs feature is disabled by default and in order to enable it, you need to add the following JSON object to a JSON configuration source.
+
+```json
+{
+  "Umbraco": {
+    "CMS": {
+      "Unattended": {
+        "InstallUnattended": true,
+        "UnattendedUserName": "FRIENDLY_NAME",
+        "UnattendedUserEmail": "EMAIL",
+        "UnattendedUserPassword": "PASSWORD"
+      }
+    }
+  }
+}
+```
+Remember to set the value of `InstallUnattended` to `true`.
+
+Alternatively you may set your configuration with Environment Variables or other means
+https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-5.0#environment-variables
+
+The keys for this would then be as follows:
+```
+Umbraco__CMS_Unattended__InstallUnattended
+Umbraco__CMS_Unattended__UnattendedUserName
+Umbraco__CMS_Unattended__UnattendedUserEmail
+Umbraco__CMS_Unattended__UnattendedUserPassword
+```
+
+## Initialize the unattended install
+
+After completing the steps above you can now initialize the installation by booting up the Umbraco instance.
+
+Once it has completed, you should see the following when visiting the frontend of the site.
+
+![Frontend of Umbraco site installed using the unattended installs feature](images/Unattended/final-screen.png)
+
+## Configuration options
+Depending on your preferences, you can use any type of configuration to specify the connection string and login information, as well as enable unattended install. Due to the new configuration functionality, it is possible to read from all kinds of sources. One example can be using a JSON file or environment variables.  
+
+**Program.cs** has a condition, which if met - *appsettings.Local.json* file will be added and configured as a configuration source. 
```

Comment:
I might be missing something here, but why exactly is this mentioned? 🤔

Reply:
Because we will be able to add the example below in this file